clcache.py

```diff
@@ -7,6 +7,7 @@
 # root directory of this project.
 #
 from ctypes import windll, wintypes
+import cProfile
```

Comment:
I tried it locally, and got

```
C:\Users\Frerich\src\clcache>python -m timeit -- "import cProfile"
1000000 loops, best of 3: 0.418 usec per loop
```

...so it seemed to be a premature optimisation.